index.js

```diff
@@ -58,7 +58,7 @@ bot.Dispatcher.on(Events.MESSAGE_CREATE, (c) => {
     Commands['newCardInit'].fn(c.message)
     return
   }
-  if (!c.message.author.bot) {
+  if (!c.message.author.bot && c.message.content.indexOf(Config.discord.prefix) !== 0) {
```

Comment:
this will trigger if i have the prefix anywhere in a message except at the beginning, you should do
`c.message.content.startsWith(Config.discord.prefix)` also you can invert that if and do
```js
if (c.message.author.bot || c.message.content.startsWith(Config.discord.prefix)) return;
```

Reply:
First point; spot on.
Second one isn’t, I haven’t checked the code for this, but returning early will kill the command incrementment